packages/account/src/Sections/Security/Passwords/passwords-static.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Localize, localize } from '@deriv/translations';
+import { Icon, Popover, Text } from '@deriv/components';
+import { connect } from 'Stores/connect';
+import DerivComLogo from '../../../Assets/ic-brand-deriv-red.svg';
+import DerivGoLight from '../../../Assets/ic-brand-deriv-go-light.svg';
+import DerivGoDark from '../../../Assets/ic-brand-deriv-go-dark.svg';
+
+const PasswordsStatic = ({ is_dark_mode_on, is_deriv_password, is_trading_password_required }) => (
+    <React.Fragment>
+        {is_deriv_password ? (
+            <div className='account__passwords-item-left'>
+                <Text className='passwords-static__text' color='less-prominent' size='xs' weight='lighter'>
+                    <Localize
+                        i18n_default_text='Use this to log in to Deriv.com, Deriv Go, DTrader,<0 /> SmartTrader, and DBot.'
+                        components={[<br key={0} />]}
+                    />
+                </Text>
+                <Text className='passwords-static__text' color='less-prominent' size='xs' weight='lighter'>
+                    {localize('Apps you have linked to this password:')}
+                </Text>
+                <div className='passwords-static__logo-container'>
+                    <DerivComLogo className='passwords-static__single-icon' />
+                    <Text line_height='l' size='xs' weight='bold'>
+                        Deriv.com
+                    </Text>
+                </div>
+                <div className='passwords-static__icons'>
+                    <Popover alignment='bottom' message={localize('DTrader')}>
+                        <Icon icon='IcBrandDtrader' size={32} />
+                    </Popover>
+                    <Popover alignment='bottom' message={localize('DBot')}>
+                        <Icon icon='IcBrandDbot' size={32} />
+                    </Popover>
+                    <Popover alignment='bottom' message={localize('SmartTrader')}>
+                        <Icon icon='IcBrandSmarttrader' size={32} />
+                    </Popover>
+                    <Popover alignment='bottom' message={localize('Deriv Go')}>
+                        {is_dark_mode_on ? <DerivGoDark /> : <DerivGoLight />}
+                    </Popover>
+                </div>
+            </div>
+        ) : (
+            <div className='account__passwords-item-left'>
+                <Text className='passwords-static__text' color='less-prominent' size='xs' weight='lighter'>
+                    {localize('Use this to log in and trade with Deriv MT5 and Deriv X.')}
```

Comment:
Should we remove the mention of Deriv X here? Since the implementation for Deriv X platform is done on a separate PR, and this might get released first

Reply:
I'm just following the design as they told me to update all DMT5 to Deriv MT5 and DXtrade to Deriv X

Reply:
Yes, but I think design is always using screens that are up to date for their end, so this includes unreleased features as well. We should clarify whether it should be included if Deriv X hasn't been released yet.